### ISSUE TITLE
POT File

### ISSUE THREAD

**Issue by shefik** [zikula-modules/InterCom#74]
Is there a POT file available for InterCom?

**Comment by Kaik** [comment]
Hi, here is old one https://github.com/zikula-modules/InterCom/blob/master/Resources/public/locale/module_intercom.pot 
I think this need to be regenerated anyway.

**Comment by shefik** [comment]
The POT file has an incorrect Project-Id-Version:

```
Project-Id-Version: Zikula 1.2.x
``

It needs to be updated to:

```
Project-Id-Version: Zikula 1.4.x
```

**Comment by Kaik** [comment]
Whole pot file needs to be regenerated old file and old directory will be deleted. This will be done after most things are finished.

**Comment by Kaik** [comment]
Included in #73 and closing.
